<ui version="4.0" >
 <class>RechteAnpassenClass</class>
 <widget class="QDialog" name="RechteAnpassenClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RechteAnpassen</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Benutzerrechte ändern</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Benutzer (ohne Administratoren):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="comboBox" />
      </item>
      <item row="1" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>neue Berechtigungen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>(mehrfachauswahl möglich)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="altCheckBox" >
          <property name="text" >
           <string>Alte Berechtigungen löschen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <widget class="QTreeWidget" name="streckenTreeWidget" >
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="canButton" >
       <property name="text" >
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
